src/components/ProfilePage/index.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import FriendsList from "./FriendsList";
+import firebaseWrapper from "../../firebaseWrapper";
+
+class ProfilePage extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      //   likedPosts: [],
+      //   userPosts: [],
+      friends: {}
+      //   activeTab: ""
+    };
+  }
+
+  componentDidMount() {
+    const { match } = this.props;
+    const { params } = match;
+    const { profileId } = params;
+    const updateFriends = friends => {
+      this.setState({
+        friends
+      });
+    };
+    firebaseWrapper.getFriends(profileId, updateFriends);
+  }
+
+  addFriend = () => {
+    const { match } = this.props;
+    const { params } = match;
+    const { profileId } = params;
+    const { userId } = this.props;
+    const addedNewFriend = friends => {
+      this.setState({
+        friends
+      });
+    };
+    const newFriendData = {};
+    newFriendData[userId] = "random";

```

Comment:
remember to not hard code this. this should be handled in firebaseWrapper with the new interface